### PR TITLE
Discussion edits

### PR DESCRIPTION
**Pull request by davharris**
The biggest change here is for the way lead times are discussed. We don't really have evidence that the models' performance was converging, so I cut a lot of material there and tried to re-organize it in a bigger-picture way.

### CODE REVIEW

**Comment by ethanwhite** [comment]
I guess I misunderstood where our conversation on this had ended up. My impression was that based on our agreement with @sdtaylor's last statement that

> different models still have different rates of error increase though, so the idea of convergence is still sound and can be discussed.

that we were all OK discussing this as a possibility.

**Comment by ethanwhite** [comment]
> Given the potential changes with lead time of the models and processes are most relevant to forecasting

Currently comes out of nowhere.

File: manuscript/draft.Rmd [hunk 1]
```diff
@@ -640,42 +633,37 @@ the habitats and species being surveyed [@sauer1994]. Accounting for differences
...
-error can be important for properly estimating and reducing uncertainty in
-forecasts and can also lead to changes in the best methods for forecasting [Box
-1]. Another key aspect of an ideal observation model is imperfect detection. In
-this study, we did not address differences in detection probability across
-species and sites [@boulinier1998] since there is no clear way to address this
-issue using NA Breeding Bird Survey data without making strong assumptions about
-the data [i.e., assuming there is no biological variation in stops along a
-route; @white2010], but this would be a valuable addition to future forecasting
-models.
+improving most models' point estimates relative to the naive baseline [Box 1].
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: This Box 1 citation looks like it is associated with "improving most models' point estimates relative to the naive baseline", which doesn't make sense to me.

File: manuscript/draft.Rmd [hunk 2]
```diff
@@ -640,42 +633,37 @@ the habitats and species being surveyed [@sauer1994]. Accounting for differences
...
 effects resulted in changes in which models performed best, most notably
-reducing the relative performance of the naive model's point estimates. This
-suggests that, prior to accounting for observer effects, the naive model
-performed well because it was capable of accommodating rapid shifts in estimated
-richness introduced by changes in the observer. These kinds of rapid changes
-were difficult for the other single-site models to accommodate, and so the
-average performance of the ARIMA and average models improved once this source of
-observation error was addressed. This demonstrates that modeling observation
-error can be important for properly estimating and reducing uncertainty in
-forecasts and can also lead to changes in the best methods for forecasting [Box
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: I like this sentence in that it ties back in to the overarching theme of the ms and reminds readers that this particular method is important to the overall goals of better forecasting. What's your thinking on the reason to take it out?

**Comment by sdtaylor** [comment]
The specific models, JSDM GBM and the average, are probably not converging. But there is still different rates of decay among all models, so perhaps this can be more generic. 

The accuracy for forecasts is expected to decline as forecasts are made further into the future, so it is 
important to evaluate forecasts at different lead times [Box 1; @petchey2015]. _For deviance (the error metric that evaluates both models' point estimates and their uncertainty simultaneously) there is different rates of decay among some models, with the stacked SDM decaying the fastest followed by the naive model. The remaining models all have similar rates of error decay. Depending on which models are evaluated this leads to the potential convergence of models over time, where models which perform worse at short lead times then perform better at longer lead times._ This potential convergence is biologically plausible because at short lead times, environmental changes are relatively small and changes in the biota may lag behind changes in the environment. However, as the lead time of the forecast increases the amount of environmental change should increase and the systems will have more time to respond, potentially resulting in shifts towards improved relative performance of models incorporating environmental and ecological information [@blonder2017]. This is potentially important for interpreting our results...

**Comment by davharris** [comment]
How's this for the paragraph we discussed? https://github.com/weecology/bbs-forecasting/pull/198/commits/3760770078aaca29c7fdbcb9e5d74f4c6508de0c

I think it might need one more sentence at the end, but it flows better and should act as a better caveat/appeasement mechanism.

**Comment by sdtaylor** [comment]
The blonder2017 ref doesn't make sense without the discussion about time lags in environmental responses. But it can probably just go with the group of refs just prior to it.

**Comment by sdtaylor** [comment]
Otherwise I like it. I did the stats on the trends as well and agree with the last statement. 

image

**Comment by davharris** [comment]
Great. I've moved the Blonder citation and will deal with Ethan's other two comments next.

File: manuscript/draft.Rmd (hunk 2 above)
Comment (davharris) [reply manuscript/draft.Rmd]: I honestly don't know why removed it. Putting it back in now..

**Comment by davharris** [comment]
Thanks guys! Ready for another review.

**Comment by ethanwhite** [comment]
Nice work @davharris! This is good for me. @sdtaylor if you're happy go ahead and merge.

**Comment by sdtaylor** [comment]
:+1:

**Pull request merged**